vpr/src/route/overuse_report.cpp

```diff
@@ -197,10 +198,10 @@ static void report_overused_chanx_chany(std::ostream& os, RRNodeId node_id) {
     //CHANX/CHANY rr nodes span across several grid locations.
     //Need to print out their starting and ending locations.
     os << "Grid location: " << '\n';
-    os << "Xlow = " << device_ctx.rr_nodes.node_xlow(node_id) << ", ";
-    os << "Ylow = " << device_ctx.rr_nodes.node_ylow(node_id) << '\n';
-    os << "Xhigh = " << device_ctx.rr_nodes.node_xhigh(node_id) << ", ";
-    os << "Yhigh = " << device_ctx.rr_nodes.node_yhigh(node_id) << '\n';
+    os << "Xlow = " << rr_graph.node_xlow(node_id) << ", ";
```

Comment:
This routine could probably use your new RRNode to string function.  Could still print extra info if the string doesn't give everything.